rllib/examples/self_play_league_based_with_open_spiel.py

```diff
@@ -31,22 +31,25 @@
 """
 
 import argparse
-import numpy as np
 import os
-from open_spiel.python.rl_environment import Environment
-import pyspiel
 import re
 
+import numpy as np
+from open_spiel.python.rl_environment import Environment
```

Comment:
Dumb question: what was the difference between `open_spiel` and `pyspiel` again? Would we need to return the `open_spiel` package as well from `try_import_pyspiel()`?

Reply:
open spiel uses pygame, which is a game engine with a python interface.

Reply:
They are separate packages though.
We list pygame in our test requirements and open_spiel nowhere.

Reply:
I have added open_spiel to this PR with the same behaviour!